src/client/rest/RESTMethods.js

```diff
@@ -377,6 +377,12 @@ class RESTMethods {
     return this.rest.makeRequest('get', Constants.Endpoints.channelMessage(channel.id, messageID), true);
   }
 
+  putGuildMember(guild, user, options) {
+    return this.rest.makeRequest('put', Constants.Endpoints.guildMember(guild.id, user.id), true, options).then(data =>
```

Comment:
Yea, I was specifically asking Gus if it actually exceeds the line limit, but in the end it boils down to that theres "too much going on in one line" so splitting it for clarity would be best.